index.bs

```diff
@@ -3375,20 +3377,6 @@ Maturity Stages on the Recommendation Track</h4>
 			in the W3C Patent Policy [[!PATENT-POLICY]]
 			for implication on patent licensing obligations.
 
-		<dt>
-			<dfn export id="RecsPR" lt="W3C Proposed Recommendation | Proposed Recommendation | PR">Proposed Recommendation</dfn> (<abbr>PR</abbr>)
```

Comment:
Would it be worth keeping a stub definition for this, in case people are looking at old PRs on /history and want to know what they are?

Reply:
The id is reserved, so that links will still land in a reasonably relevant place. That place does not mention Proposed Recs explicitly though:
https://github.com/w3c/process/pull/868/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R3885

I see your point, but at the same time, because Proposed Recs are only an transient state, it's not that common for people to be reading them once the relevant AC Review has closed.

We don't keep a stub definition for Last Call Working Draft either.

Maybe we could add an Appendix to the Process listing all the TR statuses that no longer exist, with pointers to older version of the Process that still had them? Not sure if this would be useful, or just make the Process longer for little benefit.

Reply:
Looking back through /history I see that some SOTDs include a hyperlink from the maturity stage term to its definition in the applicable Process document, e.g. [XML1.1 PR](https://www.w3.org/TR/2003/PR-xml11-20031105/) but others do not, e.g. [HTML5 PR](https://www.w3.org/TR/2014/PR-html5-20140916/). For those that do not, it would be useful to include this appendix.

Reply:
As discussed on the Process CG call last week, I've created a pull request to handle this kind of stub definitions: https://github.com/w3c/process/pull/898.
If that PR is adopted, I'll add an entry there for Proposed Rec in this PR.